website/bot.py

```
@@ -1,202 +1,182 @@
+
+    if not user_message:
+        return Response({"message": "Please type something."}, status=status.HTTP_200_OK)
+
+    safe_message = escape(user_message)
+
+    vector_store = load_vector_store()
+    if vector_store:
+        try:
+            chain = conversation_chain(vector_store)
+            result = chain.invoke({"input": safe_message})
+            answer = result.get("answer") or result.get("output") or "Sorry, I couldn’t find an answer."
```

Ruff (0.14.7)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)